src/nodejs/questions/src/schema/question.schema.ts

```diff
@@ -8,14 +8,20 @@ export class Question {
     @Prop()
     id: string;
 
+    @Prop()
+    questionId: number;
+
     @Prop()
     questionTitle: string;
 
     @Prop()
     questionCategories: Array<string>;
```

Comment:
Not sure if you'll need to indicate type explicitly for nested data structures, here's the [documentation](https://docs.nestjs.com/techniques/mongodb) for your reference.